katello/katello-backup

```diff
@@ -224,15 +270,31 @@ def offline_backup(database, dir_path = nil)
   end
 end
 
+def compress_files
+  psql = spawn('gzip', 'pgsql_data.tar') if @databases.include? "pgsql"
+  mongo = spawn('gzip', 'mongo_data.tar')
+  Process.wait(psql) if @databases.include? "pgsql"
+  Process.wait(mongo)
+end
+
+def plugin_list
+  if @is_foreman_proxy_content
+    eval(`curl -k https://$(hostname):9090/features`)
```

Comment:
@johnpmitsch I suppose thats a "safe" assumption, but the array should be valid JSON that you could parse?